### DISCUSSION TITLE
Recognize switched in maintenance Mode

### DISCUSSION THREAD

**Discussion by r3df0chs** [matiasdelellis/facerecognition#660] [General]
Maybe for someone interesting, recognize switched in maintenance Mode and no new features will be added.

https://github.com/nextcloud/recognize/discussions/779

Thank you all who contribute to FaceRecognition

**Comment by matiasdelellis** [comment]
Thank you Cebrain, I am aware of the change. I was surprised at first, but several times they went through the same procedure, announcing revolutionary products and then almost abandoning them. Likewise, as I said in other discussions, a project as big as nextcloud is difficult to maintain and they must prioritize objectives.

So, continue with the same plan. Within limitations (Mainly my limitations, time, etc.), try to make a good product, independent of other products...

In particular, I hope to have time next week to deal with some issues. 😄
